app/models/wpcc/salary_message.rb

```diff
@@ -0,0 +1,22 @@
+module Wpcc
+  class SalaryMessage
+    include ActiveModel::Model
+
+    attr_accessor :salary, :salary_frequency
+
+    TAX_RELIEF_THRESHOLD_RATE = {
+      year: 11_500,
+      month: 958.33,
+      fourweeks: 884.61,
+      week: 221.15
+    }.freeze
+
+    def salary_below_tax_relief_threshold?
+      if TAX_RELIEF_THRESHOLD_RATE.keys.exclude?(salary_frequency.to_sym)
+        return false
+      end
+
+      salary < TAX_RELIEF_THRESHOLD_RATE[salary_frequency.to_sym]
+    end
```

Comment:
Do you know, seeing this method again I would do a bit different:

```ruby
def salary_below_tax_relief_threshold?
   valid_salary_frequency? && salary < TAX_RELIEF_THRESHOLD_RATE[salary_frequency.to_sym]
end

def valid_salary_frequency?
  salary_frequency.in?(TAX_RELIEF_THRESHOLD_RATE)
end
```

Wdyt? The name valid_ could be different though. (it is the best name I can pop of my mind atm)